xbmc/music/Song.h

```diff
@@ -72,6 +72,11 @@ class CSong: public ISerializable
     return false;
   }
 
+  //Get artist names as a string vector from the vector of artist credits
```

Comment:
Sorry, but this is not what I meant, please see https://github.com/DaveTBlake/xbmc/blob/MusicInfoTagFix/xbmc/music/Artist.h#L75 as reference